stdlib/char.mc

```diff
@@ -1,3 +1,5 @@
+include "seq.mc"
```

Comment:
Why is this needed?

Reply:
A use of `head`. It works when `char.mc` is included somewhere else, since `prelude` is already included at that point, but when running tests on the standard library I run them without `prelude`, since it would induce an import cycle otherwise (well, only on the files that are included in `prelude`, but still, that's annoying to detect for the test runner).